### PR TITLE
Phazon

### PR DESCRIPTION
**Pull request by mistyLuminescence**
TL;DR:
* Readds the Phazon to Robotics. It's a mech that can walk through walls. Yeah, it's hilariously useful.
* It also has 200 health and the worst armor of any combat mech, and will die to a few shots from an ion rifle.
* You need a horrifying amount of materials and research to build it, and cooperation from multiple departments- and you can only ever make two, no matter how hard you try. (You also need a hand teleporter and a superconductive capacitance coil, of which two of each exist.)
* Construction sprites 'borrowed' from tg, because they actually have the Phazon and somehow make it work.

**MECHATRONICS BRIEF: XRM-001 'PHAZON'**

To engage in the creation of a Phazon, the following materials are required:
**300** standard-size metal sheets
**65** standard-size glass sheets
**100** gold ingots, 24Kt
**100** silver ingots
**50** uranium rods
**50** phoron crystals
**10** osmium bars
**40** diamonds

After conducting **sufficient research**, place the materials in your **exosuit fabricator**, printing off the individual Phazon parts and connecting them together as per *Mechatronics For Dummies: Robots And You*.

After installing the **circuitry** (available from **the Supply department** for **500 points** - note that the crate requires the authorisation of the **Research Director or Overseer**), take a **superconductive capacitance coil** and connect both ends to the chassis, installing the coil in the marked slot.

Place a **handheld teleporter** inside the chassis, connecting the antenna to the superconductive capacitance coil. You may now proceed with armoring the Phazon, using **steel** and **plasteel**.

The Phazon has the following unique capabilities:
- Phasic capacitance field: The Phazon can project a bluespace field around it, using the Hans-Friedrich Singulum to shunt the mech, pilot and passengers out of shift with reality for a brief time. This can be used to travel through solid objects such as walls. The phasic capacitance field consumes power for each use. **Use of the phasic capacitance field to gain access to restricted areas is grounds for termination.**
- Adaptive biothermic underlay: The Phazon can alter its arm extensions to coat them in either a highly toxic phorogenic substance, a superheated oxyacetylene vapor. Along with standard configuration, the Phazon is capable of dealing brute, toxin or burn damage with its punches. **Use of the adaptive biothermic underlay is a violation of several interstellar weapons treaties, and is grounds for termination.**

Due to the Phazon being spatially desynchronised, it is more fragile than other mechs, and should not be used as a combat mech. A salvo of well-placed ionic discharges will destroy the mech and render it permanently nonfunctional.

Please note that these documents are purely hypothetical, as technology has not progressed to a stage where construction of the Phazon would be possible.

### CODE REVIEW

**Comment by mistyLuminescence** [comment]
"Misty. Misty, why."

*Because we can.*

And also because Science needs something to aim for in those 3+ hour rounds that isn't "let's build the break room out of diamonds". And what better than making a scary-fast noclipping mech? It's a pretty nice PR for #6000. Sure, it might not see much use, but that's the point. It's something to work towards. And since you need cooperation from Cargo, Engineering, Mining, Command, R&D and Robotics... well, if you can actually make one of these, you deserve it.

And if you can make two, I'm in awe, but you're a monster who just used both hand teles on some snazzy mechs. Just put a passenger compartment on it like a normal person, dammit.

**Comment by Atermonera** [comment]
`"Because we can"` is never a valid reason to give players, through _any_ mechanism, adminspawn-only items, _especially_ without consulting _any_ staff whatsoever prior to writing the code.

FURTHERMORE, if something comes with a feature that has the tagline "You will actually get fired for using this and possibly tried for some manner of crime related to the use of this", then producing such a thing will similarly be a fire-able offense.

**Comment by mistyLuminescence** [comment]
1) I told Mechoid that I was making it in the discord. Staff were consulted.
2) Read the part that comes *after* the "because we can"- it's really because Science lacks any kind of ultra-expensive endgame project, and with a decent Mining team, everything they can produce, they can mass-produce for every member of the crew three times over. If you're somehow getting a bunch of different departments to cooperate with you, you deserve a reward, preferably one that can walk through walls.
3) You can produce a ton of stuff without being fired, you just can't use them. Laser cannons, armed Gygaxes and Durands, fuel rod guns, slime armies that could conceivably take over the station... you can't really use any of these outside of antagging or really bad emergencies, and it's technically possible to use a Phazon without breaking any laws as long as you don't use the biopunch or noclip into the CD's office or something.

**Comment by mistyLuminescence** [comment]
Ideas so far to make it require more departmental cooperation, and have it be less useful:
* Change metal and plasteel (for armor) into deuterium and durasteel respectively, requiring cooperation from Chemistry to finish.
* Remove biopunch, AKA war crime melee, and make an adminspawn-only variant with the original armor (0.7/30% DR) and biopunch.
* If we want to be real brutal, only give it one weapon slot (or none, like the Odysseus), with the regular amount being on the adminspawn variant.
